File: Programmers/src/Javascript/Lv1/콜라문제.js
function solution(a, b, n) {
    var answer = 0;
    let remain = n;
    
    while(true){
        if(a > remain)  break;
        
        answer += parseInt(remain/a)*b;
        remain = parseInt(remain/a)*b + remain%a;
    }
    
    return answer;
}

File: Programmers/src/Java/Lv1/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;
        
        while(true) {
            if(n<a)
                break;
            
            answer += b * (n/a);
            n = (n/a)*b + n%a;
        }
        
        return answer;
    }
}